install_config/install/disconnected_install.adoc

```diff
@@ -259,28 +253,37 @@ xref:../../architecture/core_concepts/builds_and_image_streams.adoc#source-build
 (S2I)] builder images that you intend to use in your OpenShift environment. You
 can pull the following images:
 +
-* jboss-eap70-openshift
-* jboss-amq-62
-* jboss-datagrid65-openshift
-* jboss-decisionserver62-openshift
-* jboss-eap64-openshift
-* jboss-eap70-openshift
-* jboss-webserver30-tomcat7-openshift
-* jboss-webserver30-tomcat8-openshift
-* jenkins-1-rhel7
-* jenkins-2-rhel7
-* jenkins-slave-base-rhel7
-* jenkins-slave-maven-rhel7
-* jenkins-slave-nodejs-rhel7
-* mongodb
-* mysql
-* nodejs
-* perl
-* php
-* postgresql
-* python
-* redhat-sso70-openshift
-* ruby
+----
+# docker pull registry.access.redhat.com/jboss-eap-7/eap70-openshift
```

Comment:
I think `# docker pull registry.access.redhat.com/jboss-eap-7/eap70-openshift` is listed twice. Otherwise, LGTM! 🍂

Reply:
I found a few repeats! Such a long list. Thanks!